src/test/java/jewellery/inventory/helper/ProductTestHelper.java

```diff
@@ -50,7 +50,7 @@ public static ProductRequestDto getProductRequestDto(
     productRequestDto.setCatalogNumber("1111");
     productRequestDto.setProductionNumber("1234");
     productRequestDto.setDescription("Test");
-    productRequestDto.setAdditionalPrice(BigDecimal.TEN);
+    productRequestDto.setAdditionalPrice(null);
```

Comment:
I guess having an additional price breaks the tests. You should either take it into account when asserting for the end price or make it 0. Let's not allow null as an option